clients/node/src/clients/base.ts

```diff
@@ -0,0 +1,111 @@
+import { ClientConfig } from '../models/clientConfig';
+const crypto = require('crypto');
+import * as ed from '@noble/ed25519';
+import * as bs58 from 'bs58';
+import { ApiVersion } from '../models/apiVersion';
+import axios, { AxiosInstance } from 'axios';
+
+/**
+ * This is the base client used as a parent class for all clients
+ * using the integration services api.
+ */
+export abstract class BaseClient {
+  private apiKey: string;
+  private baseUrl: string;
+  private apiVersion: ApiVersion;
+  public jwtToken?: string;
```

Comment:
```suggestion
  protected jwtToken?: string;
```
protected should be enough since the jwt is only used in children classes

Reply:
This is wrong: jwtToken need to be private and be used only in the Base class. I'll change methods removing jwtToken as parameter